src/components/atoms/socials/socials.tsx

```diff
@@ -0,0 +1,43 @@
+import './socials.scss';
+
+type Props = {
+  emailUrl: string,
+  facebookUrl: string,
+  twitterUrl: string,
+  linkedinUrl: string,
+  redditUrl: string,
+};
+
+export const Socials = ({
+  emailUrl, twitterUrl, facebookUrl, linkedinUrl, redditUrl,
+}: Props): JSX.Element => (
+  <div className="socials-container">
+    <ul className="socials-sharers">
+      <li>
```

Comment:
Would it be better to make these optionals, then only display the ones we have passed a URL for?